piccolo/testing/model_builder.py

```diff
@@ -139,11 +139,18 @@ def _randomize_attribute(cls, column: Column) -> t.Any:
             Column class to randomize.
 
         """
+        random_value: t.Any
         if column.value_type == Decimal:
             precision, scale = column._meta.params["digits"]
             random_value = RandomBuilder.next_float(
                 maximum=10 ** (precision - scale), scale=scale
             )
+        elif column.value_type == list:
+            length = RandomBuilder.next_int(maximum=10)
+            base_type = t.cast(Array, column).base_column.value_type
```

Comment:
is `Array` the only column type based on lists?

Reply:
Yep, Array is the only one with `value_type = list`.